connector/sccconnector/factory.go

```diff
@@ -0,0 +1,106 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sccconnector // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/sccconnector"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/connector"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.uber.org/zap"
+)
+
+// this is the name used to refer to the connector in the config.yaml
+const (
+	typeStr      = "scc"
+	scopeName    = "otelcol/sccconnector"
+	scopeVersion = "v0.0.1"
+	schemaUrl    = "https://opentelemetry.io/schemas/1.26.0"
+)
+
+// NewFactory creates a factory for example connector.
+func NewFactory() connector.Factory {
+	// OpenTelemetry connector factory to make a factory for connectors
+	return connector.NewFactory(
+		component.MustNewType(typeStr),
+		createDefaultConfig,
+		connector.WithTracesToLogs(createTracesToLogsConnector, component.StabilityLevelAlpha),
```

Comment:
You'll want to use development stability here for all three connector types, to match `metadata.yaml`.

Reply:
Good catch.  I updated to use the generated metadata.